### DISCUSSION TITLE
Use the core filter?

### DISCUSSION THREAD

**Discussion by mattradford** [wp-privacy/wp-api-privacy#17] [General]
Hi @duanestorey. Thanks for writing this plugin. I wrote [a very simple plugin when WP 4.9 was released](https://wordpress.org/plugins/update-privacy/) that used the `core_version_check_query_args` filter. I'm curious why you're not using this in the `modifyCurl()` method?
Thanks, Matt

**Comment by duanestorey** [comment]
Hey thanks. I actually wasn't aware this function exists.   I tried originally to get at the URL through the HTTP code, but it didn't have any good hooks.  I'll take a look at that one, thanks.

**Comment by duanestorey** [comment]
I updated to use this method, thanks.  It's much cleaner now.
